_includes/disqus.html

```diff
@@ -0,0 +1,12 @@
+{% if page.comments %}
+<div id="disqus_thread"></div>
+<script>
+(function() { // DON'T EDIT BELOW THIS LINE
+var d = document, s = d.createElement('script');
+s.src = 'https://solar-forecast-arbiter.disqus.com/embed.js';
+s.setAttribute('data-timestamp', +new Date());
+(d.head || d.body).appendChild(s);
+})();
+</script>
+<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
+{% endif %}
```

Comment:
I thought we were dropping disqus for now.

Reply:
Yes, I forgot to remove the includes, but removed the includes statements. Fixed this.